rest-api-json-bulk/src/main/kotlin/BulkApi.kt

```diff
@@ -47,8 +49,8 @@ fun Route.BulkApi(loadRoots: suspend () -> List<INode>, resolve: suspend (INodeR
         val roomList = LectureList(lectures = allLectures.map {
             Lecture(name = it.properties.name!!,
                     description = it.properties.description!!,
-                    lectureRef = INodeReferenceSerializer.serialize(it.iNode.reference),
-                    room = INodeReferenceSerializer.serialize(it.references.room.iNode.reference),
+                    lectureRef = URLEncoder.encode(INodeReferenceSerializer.serialize(it.iNode.reference), Charset.defaultCharset()),
```

Comment:
Maybe so convenience function a la `refToRest` to avoid having to repeat (and forget) the URL encoding every time?